string  obj
---
object_detection/obj_detection_msg data